wallet/how-to/register-token.md

```diff
@@ -3,17 +3,37 @@ description: Register a token with users using wallet_watchAsset.
 sidebar_position: 6
 ---
 
-# Register a token with users
+# Display tokens
 
-When a user opens MetaMask, they're shown a variety of assets, including tokens.
-By default, MetaMask detects some major popular tokens and displays them, but for most tokens, the
-user must register the token themselves.
-This process can be cumbersome, involves the user interacting with contract addresses, and is error-prone.
+When a user opens MetaMask, they're shown some major tokens by default.
+However, to display most custom ERC-20 tokens, the user must [add the token
+manually](https://support.metamask.io/hc/en-us/articles/360015489031-How-to-display-tokens-in-MetaMask#h_01FWH492CHY60HWPC28RW0872H).
+To display a user's NFTs, the user must [explicitly turn on NFT autodetection or add their NFTs
+manually](https://support.metamask.io/hc/en-us/articles/360058238591-NFT-tokens-in-your-MetaMask-wallet).
 
-You can improve the security and experience of users registering your token on their MetaMask
-interface by using the [`wallet_watchAsset`](../reference/rpc-api.md#wallet_watchasset) RPC method.
+Manually adding tokens involves the user interacting with contract addresses, and is error-prone.
 
-For example, you can add something like the following to your project script:
+You can improve the security and experience of displaying your
+[ERC-20 token](#register-an-erc-20-token) or users' [NFTs](#register-nfts) in MetaMask by using the
+[`wallet_watchAsset`](../reference/rpc-api.md#wallet_watchasset) RPC method.
```

Comment:
this should link to the API Playground method: `../reference/wallet_watchasset/`